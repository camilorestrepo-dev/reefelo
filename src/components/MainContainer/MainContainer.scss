@import "../../styles/shared.scss";

.page-container {
  min-height: 100vh;
  min-width: 100vw;
  background-color: $white;
  color: $text-color;
  display: flex;
  justify-content: flex-start;
  align-items: center;
  flex-direction: column;

  .main-continer {
    margin: $spacing-24 auto;
    min-width: 320px;
    max-width: 1200px;
    width: 90%;
  }
}