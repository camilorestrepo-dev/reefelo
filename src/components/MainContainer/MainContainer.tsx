import "./MainContainer.scss";

const MainContainer: React.FunctionComponent<React.PropsWithChildren> = ({
  children,
}: React.PropsWithChildren) => {
  return (
    <div className="page-container">
      <main className="main-continer">{children}</main>
    </div>
  );
};

export default MainContainer;
