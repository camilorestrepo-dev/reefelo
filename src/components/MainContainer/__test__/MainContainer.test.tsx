import { render } from "@testing-library/react";
import MainContainer from "../MainContainer";

test("reder corectly", () => {
  const component = render(
    <MainContainer>
      <div>Some content</div>
    </MainContainer>,
  );
  expect(component.container).toMatchSnapshot();
});
