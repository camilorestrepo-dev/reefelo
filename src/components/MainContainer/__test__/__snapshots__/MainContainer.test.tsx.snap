// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reder corectly 1`] = `
<div>
  <div
    class="page-container"
  >
    <main
      class="main-continer"
    >
      <div>
        Some content
      </div>
    </main>
  </div>
</div>
`;
